README.md

```diff
@@ -1,78 +1,204 @@
 
-You can find an example project [here](https://github.com/uploadcare/pyuploadcare-example).
+Assume you have a Django project with gallery app.
+
+Add `pyuploadcare.dj` into `INSTALLED_APPS`:
 
 ```python
+INSTALLED_APPS = (
+    # ...
+    'pyuploadcare.dj',
+
+    'gallery',
+)
+```
+Add API keys to your Django settings file:
 
-    from django import forms
-    from django.db import models
+```python
+UPLOADCARE = {
+    'pub_key': 'demopublickey',
+    'secret': 'demoprivatekey',
+}
+```
 
-    from pyuploadcare.dj.models import ImageField
-    from pyuploadcare.dj.forms import FileWidget, ImageField as ImageFormField
+Uploadcare image field adding to your `gallery/models.py` is really simple. Like that:
 
+```python
+from django.db import models
 
-    class Candidate(models.Model):
-        photo = ImageField(blank=True, manual_crop="")
+from pyuploadcare.dj.models import ImageField
 
 
-    # optional. provide advanced widget options: https://uploadcare.com/docs/uploads/widget/config/#options
-    class CandidateForm(forms.Form):
-        photo = ImageFormField(widget=FileWidget(attrs={
-            'data-cdn-base': 'https://cdn.super-candidates.com',
-            'data-image-shrink': '1024x1024',
-        }))
+class Photo(models.Model):
 
+    title = models.CharField(max_length=255)
+    photo = ImageField()
 ```
 
-![](https://ucarecdn.com/f0894ef2-352e-406a-8279-737dd6e1f10c/-/resize/800/manual_crop.png)
+`ImageField` doesn’t require any arguments, file paths or whatever. **It just works**. That’s the point of it all. It looks nice in the admin interface as well:
 
-## Documentation
+<!-- TODO: Update screenshot -->
+![](https://ucarecdn.com/84e614e4-8faf-4090-ba3a-83294715434b/)
+
+Obviously, you would want to use Uploadcare field outside an admin. It’s going to work just as well, but, however, you have to remember to add {{ form.media }} in the <head> tag of your page:
+
+```htmldjango
+{{ form.media }}
+
+<form action="" method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <input type="submit" value="Save"/>
+</form>
+```
+
+This is a default Django form property which is going to render any scripts needed for the form to work, in our case — Uploadcare scripts.
+
+<!-- TODO: Update how we get an UUID and pass it further -->
+After an image is uploaded, you can deliver it while transforming it on the fly:
+
+```htmldjango
+{% for photo in photos %}
+    <h2>{{ photo.title }}</h2>
+    <img src="{{ photo.photo.cdn_url }}-/resize/400x300/-/effect/flip/-/effect/grayscale/">
+{% endfor %}
+```
+
+<!-- TODO: Maybe we should link to the URL builder pyuploadcare docs? -->
+(Refer to Uploadcare [image processing docs](https://uploadcare.com/docs/transformations/image/) for more information).
+
+<!-- TODO: Connect this example to the rest of them -->
```

Comment:
It doesn't really demonstrate anything new here (apart from the widget configuration — but that could be a part of the docs).